mangle-shebangs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+SYSTEM_RELEASE_PATH=/etc/system-release
+
+# RHEL8 and Fedora30 both treat shebangs of the form #!/usr/bin/env
+# python as errors, and both default to python2
+if [ -f $SYSTEM_RELEASE_PATH ] && [[ "$(cat $SYSTEM_RELEASE_PATH)" =~ Red\ Hat\ Enterprise\ Linux\ release\ 8|Fedora\ release\ 30 ]]; then
+    # Replace the first line in .py to "#!/usr/bin/env python2" no matter what it was before
+    sed -i -e '1 s/^.*$/\#!\/usr\/bin\/env python2/' src/watchdog/__init__.py
+    sed -i -e '1 s/^.*$/\#!\/usr\/bin\/env python2/' src/mount_efs/__init__.py
+fi
+
+# Fedora31+ treats shebangs of the form #!/usr/bin/env python as
+# errors, and it defaults to python3
+if [ -f $SYSTEM_RELEASE_PATH ] && [[ "$(cat $SYSTEM_RELEASE_PATH)" =~ Fedora\ release\ 3[1-9]|[4-9][0-9] ]]; then
+    # Replace the first line in .py to "#!/usr/bin/env python3" no matter what it was before
+    sed -i -e '1 s/^.*$/\#!\/usr\/bin\/env python3/' src/watchdog/__init__.py
+    sed -i -e '1 s/^.*$/\#!\/usr\/bin\/env python3/' src/mount_efs/__init__.py
+fi
```

Comment:
Since we require python2 installation, I would recommend keep the same way for Fedora31, 32, 33 and Fedora 30. That is, change the shebang to python2.

Reply:
See commit 7f4d4c6.